index.html

```diff
@@ -5,17 +5,20 @@
     <title>Learn Heroku - Hello World!</title>
   </head>
   <body>
-    <h1>Hello World!</h1>
-<style>
-h1 {
-  color: teal;
-  border: 3px teal solid;
-  font-size: 6em;
-  padding: 1em;
-  margin: 0.5em;
-  text-align: center;
-  font-family: sans-serif;
-}
-</style>
+    <div style="background-color:#f0f0f5;color:#006699;padding:20px;">
+    <p>
+        Dear employee,
+    </p>
+    <p>
+    You have been chosen to provide feedback for you manager. Please answer the survey questions honestly.
```

Comment:
@AGegen stoked that you are learning using this tutorial. 🎉 
but this is _probably_ not relevant to the other people using this...? 🤔

Reply:
Oh hi, yes I didn't mean to do that. I'm sorry. Thank you for the great tutorial!